recipes/retrying/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: retrying
+  version: "1.3.3"
+
+source:
+  fn: retrying-1.3.3.tar.gz
+  url: https://pypi.python.org/packages/source/r/retrying/retrying-1.3.3.tar.gz
+  md5: 2a126aeef8b21324ecdeac15ff46ef17
+
+build:
+  noarch_python: True
+  number: 1
+
+requirements:
+  build:
+    - python
+    - six >=1.7.0
+  run:
+    - python
+    - six >=1.7.0
+
+test:
+  imports:
+    - retrying
+
+about:
+  home: https://github.com/rholder/retrying
+  license: Apache Software License
+  summary: 'Retrying'
```

Comment:
Can you add a more detailed summary? Maybe the first sentence from the github page?